assets/tests/localStorage.test.ts

```diff
@@ -0,0 +1,75 @@
+import { filter, loadState, saveState } from "../src/localStorage"
+
+const APP_STATE_KEY = "mbta-skate-state"
+
+const mockLocalStorage = {
+  getItem: jest.fn(),
+  setItem: jest.fn(),
+}
+Object.defineProperty(window, "localStorage", {
```

Comment:
Do we need to reset this when the test is done to avoid polluting other tests?

Reply:
Can't hurt